### ISSUE TITLE
Add ability to import the constants of the action-types

### ISSUE THREAD

**Issue by ondrasak** [neilff/redux-ui-router#90]
Would be great to have ability to import the constants of the action-types .

**Use case:**
I want to listen on some action and perform side-effect
now I need to hardcode the name of the action

**Comment by hally9k** [comment]
@ondrasak I'm not sure that I fully understand your proposal. Could explain it in a bit more detail?

**Comment by ondrasak** [comment]
let say I want the behaviour:
When route/state is changed -> close side navigation

so in some redux-middleware (i my case redux-observable) -> I will check all action if their type is equal to '@@reduxUiRouter/$stateChangeStart' -> if YES then perform desired action

so I need to get somehow the string "@@reduxUiRouter/$stateChangeStart"

would be great to have ability to import that constant
```ts
import {STATE_CHANGE_START} from "redux-ui-router"
// or
import {actionTypes} from "redux-ui-router" // -> actionTypes.STATE_CHANGE_START
```

**Comment by hally9k** [comment]
I understand now. As opposed to maintaining those constants externally. I may get round to this in a week or so but open to a PR in the meantime 👍

**Comment by skortchmark9** [comment]
Yes, agreed I have to redefine the ui router constants in my app.

**Comment by skortchmark9** [comment]
Actually, a slightly more stable hack than defining them yourselves is to use the action creators:
```javascript
const goAction = stateGo().type
```

**Comment by skortchmark9** [comment]
Incidentally, you might want to change your namespacing - it breaks https://github.com/wix/redux-saga-tester/blob/master/src/SagaTester.js#L45
